lib/samples/show_device_location_history/show_device_location_history_sample.dart

```diff
@@ -26,26 +26,39 @@ class ShowDeviceLocationHistorySample extends StatefulWidget {
   const ShowDeviceLocationHistorySample({super.key});
 
   @override
-  State<ShowDeviceLocationHistorySample> createState() =>
-      _ShowDeviceLocationHistorySampleState();
+  State<ShowDeviceLocationHistorySample> createState() => Configure();
 }
 
-class _ShowDeviceLocationHistorySampleState
-    extends State<ShowDeviceLocationHistorySample> with SampleStateSupport {
+class Configure extends State<ShowDeviceLocationHistorySample>
```

Comment:
I think we should leave the name as `_ShowDeviceLocationHistorySampleState` instead of `Configure`, so it's consistent with the other samples.